### DISCUSSION TITLE
My school dose not appear on the map!!

### DISCUSSION THREAD

**Discussion by Mohammed-Nazar** [community/community#48370] [GitHub Education]
Hi, 
Am High school student and i have an issue which is my school dose not appear on the map even in search and i get rejected for that reason, 
school name: sava private high school.
school location on google map: https://goo.gl/maps/uFwqDnT78pQ7AKU39

**Comment by samus-aran** [comment]
Hi @Mohammed-Nazar 👋 

If your school is not available on the application's map you are still able to identify your school within the application form. However we may ask for additional evidence of the school attending to verify your status.

Let us know how you get on 👍
